src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3192,6 +3204,33 @@ export class ResultsViewPageStore {
         []
     );
 
+    @computed get referenceGenomeBuild() {
+        if (this.studies.isComplete === false) {
```

Comment:
Should we just do this instead?
```suggestion
        if (!this.studies.isComplete) {
```